tests/test_plan_refsols/correl_13.txt

```diff
@@ -1,10 +1,10 @@
 ROOT(columns=[('n', n)], orderings=[])
  PROJECT(columns={'n': agg_0})
   AGGREGATE(keys={}, aggregations={'agg_0': COUNT()})
-   FILTER(condition=DEFAULT_TO(agg_1, 0:int64) > 0:int64, columns={'account_balance': account_balance})
-    JOIN(conditions=[t0.key == t1.supplier_key], types=['left'], columns={'account_balance': t0.account_balance, 'agg_1': t1.agg_1})
-     FILTER(condition=nation_key <= 3:int64, columns={'account_balance': account_balance, 'key': key})
-      SCAN(table=tpch.SUPPLIER, columns={'account_balance': s_acctbal, 'key': s_suppkey, 'nation_key': s_nationkey})
+   FILTER(condition=DEFAULT_TO(agg_1, 0:int64) > 0:int64, columns={})
```

Comment:
For my understanding, a NULL column will not show up in the plans?

Reply:
It's not a NULL column but an empty list of columns. We added a Null column in the SQLGlot AST.